src/schemas/Config.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+/**
+ * Connector configuration schema.
+ */
+export class Config {
+  /**
+   * ILP address of the connector. This property can be omitted if an account with `relation=parent` is configured under `accounts`.
+   */
+  ilpAddress?: string;
+  /**
+   * Description of connector account plugins.
+   */
+  accounts: {
+    /**
+     * Description of individual account.
+     */
+    [k: string]: {
+      /**
+       * Relationship between the connector and the counterparty that the account is with.
+       */
+      relation: "parent" | "peer" | "child";
```

Comment:
Do you think it would be more or less clear to adopt IP's terminology of upstream/downstream?